qiskit_ibm/experiment/ibm_experiment_service.py

```diff
@@ -127,6 +128,9 @@ def create_experiment(
             metadata: Experiment metadata.
             experiment_id: Experiment ID. It must be in the ``uuid4`` format.
                 One will be generated if not supplied.
+            parent_experiment_uuid: The ``uuid`` of the parent experiment.
+                The parent experiment must exist, must be on the same device as the child,
```

Comment:
```suggestion
                The parent experiment must exist, must be on the same backend as the child,
```
For consistency we use `backend` instead of `device` throughout ibmq.